packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -73,20 +74,20 @@ function GridFilterForm(props: GridFilterFormProps) {
   const ownerState = { classes: rootProps.classes };
   const classes = useUtilityClasses(ownerState);
 
-  const getCurrentColumn = React.useCallback(() => {
+  const currentColumn = React.useMemo<GridStateColDef | null>(() => {
```

Comment:
I'm not sure we actually need to memoize that since where it is being called it is already is memoized and they both depend on `item`.